ios/RCTWebRTC/WebRTCModule+DailyDevicesManager.m

```diff
@@ -161,6 +173,9 @@ - (BOOL)isBuiltInMic:(NSString*)portType {
 RCT_EXPORT_METHOD(setAudioDevice:(NSString*)deviceId) {
     NSLog(@"[Daily] setAudioDevice: %@", deviceId);
     
+    [self setAudioMode: AUDIO_MODE_USER_SPECIFIED_ROUTE];
```

Comment:
we might have a weird cycle here.

`setAudioMode:` (in `WebRTCModule+Daily`) invokes `setAudioDevice:`, and `setAudioDevice:` invokes `setAudioMode:`...

Reply:
I did a refactor, to isolate this part of the audioMode in a different category, this way we prevent any kind of cyclic dependency between the `WebRTCModule+Daily` and `WebRTCModule+DailyDevicesManager`.

Instead, now, both of them are depending on `WebRTCModule+DailyAudioMode`.

Another improvement, now that we have more than one Daily file for iOS, would be to rename from `WebRTCModule+Daily` to `WebRTCModule+DailyAudioManager`. This way would also match the name that we already use on Android.

Let me know what you think about those changes.